aikau/src/main/resources/alfresco/renderers/AvatarThumbnail.js

```diff
@@ -112,7 +162,20 @@ define(["dojo/_base/declare",
             this.currentItem[this.userNameProperty] = encodeURIComponent(this.currentItem[this.userNameProperty]);
             this.thumbnailUrlTemplate = "slingshot/profile/avatar/{" + this.userNameProperty + "}/thumbnail/avatar";
          }
+         
          this.inherited(arguments);
+
+         if (this.groupProperty && this.groupValues)
```

Comment:
Worth adding `&& this.groupValues.length` to avoid running the condition if there's no values? (Not entirely sure of the logic here)